app/src/androidTest/java/com/willowtree/vocable/tests/BaseTest.kt

```diff
@@ -46,10 +48,10 @@ open class BaseTest {
         // Since the build machine gets wiped after every run we can check the file storage
         // of the emulator to determine if this is a first time launch
         // and dismiss the full screen immersive popup on android if it is
-        firstLaunch = getInstrumentation().targetContext.filesDir.listFiles().isEmpty()
+/*        firstLaunch = getInstrumentation().targetContext.filesDir.listFiles().isEmpty()
         if (firstLaunch) {
             takeScreenshot("InitialSetup")
-        }
+        }*/
```

Comment:
Is this intentionally commented out? If so, we should delete the code. We can bring it back through git if we need to.

Reply:
That was part of a different problem. Removed.